xcube/core/gen2/local/transformer.py

```diff
@@ -0,0 +1,65 @@
+# The MIT License (MIT)
+# Copyright (c) 2021 by the xcube development team and contributors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+# of the Software, and to permit persons to whom the Software is furnished to do
+# so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from abc import ABC, abstractmethod
+from typing import Tuple
+
+import xarray as xr
+
+from xcube.core.gridmapping import GridMapping
+from .helpers import is_empty_cube, strip_cube
+from ..config import CubeConfig
+
+TransformedCube = Tuple[xr.Dataset, GridMapping, CubeConfig]
```

Comment:
No, for 3 reasons:

1. we should not name types after their Python types. 
2. I want to be able to turn this into a class later without name change
3. this is an implementation detail, the package is ment to be API

Reply:
My problem with the name is that I just a expect a single object, like a dataset. Therefore I find the name misleading. However, that would be solved by turning it into a class, so that argument convinces me.